src/lib/Components/Bidding/Screens/BillingAddress.tsx

```diff
@@ -85,55 +110,113 @@ export class BillingAddress extends React.Component<BillingAddressProps, Billing
     this.props.navigator.pop()
   }
 
+  selectNextInput = nextComponent => {
+    const inputs = [
+      this.fullName,
+      this.addressLine1,
+      this.addressLine2,
+      this.city,
+      this.stateProvinceRegion,
+      this.postalCode,
+    ]
+
+    for (const input of inputs) {
+      if (input === nextComponent) {
+        input.root.focus()
+      } else {
+        input.root.blur()
+      }
+    }
+  }
+
+  createCustomRef(refName: string, component: any) {
+    if (component) {
+      return (this[refName] = component)
+    }
+  }
+
   render() {
+    // TODO: Remove this once React Native has been updated
+    const isPhoneX = Dimensions.get("window").height === 812 && Dimensions.get("window").width === 375
+    const defaultVerticalOffset = isPhoneX ? 30 : 15
+
     return (
       <BiddingThemeProvider>
-        <ScrollView>
-          <Container>
-            <Title mt={0} mb={6}>
-              Your billing address
-            </Title>
-
-            <StyledInput
-              label="Full name"
-              placeholder="Enter your full name"
-              autoCapitalize="words"
-              {...this.propsForInput("fullName")}
-            />
-
-            <StyledInput
-              label="Address line 1"
-              placeholder="Enter your street address"
-              autoCapitalize="words"
-              {...this.propsForInput("addressLine1")}
-            />
-
-            <StyledInput
-              label="Address line 2 (optional)"
-              placeholder="Enter your apt, floor, suite, etc."
-              autoCapitalize="words"
-              {...this.propsForInput("addressLine2")}
-            />
-
-            <StyledInput label="City" placeholder="Enter city" {...this.propsForInput("city")} />
-
-            <StyledInput
-              label="State, Province, or Region"
-              placeholder="Enter state, province, or region"
-              autoCapitalize="words"
-              {...this.propsForInput("state")}
-            />
-
-            <StyledInput
-              label="Postal code"
-              placeholder="Enter your postal code"
-              autoCapitalize="words"
-              {...this.propsForInput("postalCode")}
-            />
-
-            <Button text="Add billing address" onPress={() => this.onSubmit()} />
-          </Container>
-        </ScrollView>
+        <KeyboardAvoidingView behavior="padding" keyboardVerticalOffset={defaultVerticalOffset} style={{ flex: 1 }}>
```

Comment:
This `KeyboardAvoidingView` is what gives us the space below the button so it will scroll above the keyboard.